.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "assets/Font-Awesome"]
+	path = assets/Font-Awesome
+	url = https://github.com/FortAwesome/Font-Awesome.git
+[submodule "assets/bootstrap-sass"]
+	path = assets/bootstrap-sass
+	url = https://github.com/twbs/bootstrap-sass
```

Comment:
There's two types of files in the ```assets/cache/``` directory:

* Downloaded files, such as _bootstrap.scss (if you use the example from the cookbook instead of submodules)
* Processed asset files.

Downloaded files _must_ be in git to make sure that any other developer have the same files locally. A file that exists in ```cache/``` will not be downloaded again, which makes sure that developer X has the same base as developer Y. It also helps the pipe's process() step, since it can calculate based on the correct input files.